downloads/install.sh

```diff
@@ -89,12 +89,28 @@ case $UNAME_ARC in
     ;;
 esac
 
+# If OS uses musl, set distinct OS type to ensure the binary
+# with statically linked libraries and dependencies is used
+if command -v ldd >/dev/null 2>&1 && [[ "$(ldd /bin/ls | grep -m1 'musl')" ]]; then
+    OSTYPE="linux-musl"
+fi
+
 case $OSTYPE in
 "linux-gnu"*)
     OS="linux"
+    STATIC="false"
```

Comment:
I think you can drop this `linux-gnu` block entirely now right? The `linux*` below should catch it.

Reply:
Good point. Removed it